package.json

```diff
@@ -57,15 +57,18 @@
   },
   "dependencies": {
     "3d-view": "^2.0.0",
-    "@plotly/d3-sankey": "^0.5.1",
     "alpha-shape": "^1.0.0",
     "array-range": "^1.0.1",
     "canvas-fit": "^1.5.0",
     "color-normalize": "^1.3.0",
     "convex-hull": "^1.0.3",
     "country-regex": "^1.1.0",
     "d3": "^3.5.12",
+    "d3-sankey": "git://github.com/antoinerg/d3-sankey.git#4f37ed8d3578b545a8569ecd74583f373768e900",
```

Comment:
The only appreciable change we have with d3-sankey is:
https://github.com/d3/d3-sankey/compare/master...antoinerg:fix-large-padding#diff-968964796da46b424b314b4359ccde31

Reply:
Yes, thank you @archmoj! The only changes I made are are in `src/sankey.js`.